obj.js

```diff
@@ -0,0 +1,54 @@
+const bookSection = document.querySelector('.displayed-books');
+const titleInput = document.querySelector('.title-input');
+const authorInput = document.querySelector('.author-input');
+const addBook = document.querySelector('.add-book');
+
+const LOCAL_STORAGE_KEY = 'items';
+
+const getItems = () => {
+  const savedBooks = window.localStorage.getItem(LOCAL_STORAGE_KEY);
+  return savedBooks ? JSON.parse(savedBooks) : [];
+};
+
+let books = getItems();
+
+const saveLocalStorage = () => {
+  window.localStorage.setItem(LOCAL_STORAGE_KEY, JSON.stringify(books));
+};
+
+const removeBook = (index) => {
+  books = books.filter((book, id) => index !== id);
+};
+// add clases
+// funtion create new book
```

Comment:
- [ ] Great job working on this milstone. Keeping commented pieces of codes on our codebase is not profesional. With Git, you can always view lines of deleted codes. Kindly remove all commented codes always from your codebase before pushing to Github